plugins/openai_panel.py

```diff
@@ -138,7 +138,7 @@ def preview(self, text: str) -> str | sublime.Html:
     def list_items(self) -> List[Tuple[str, Value]]:
         logger.debug('list_items _name: %s', self._name)
         if self._name == 'model':
-            return [(assistant['name'], assistant) for assistant in self.assistants]
+            return [(assistant['name'], self.check_default_values(assistant)) for assistant in self.assistants]
```

Comment:
This changes were breaking the model pick pane (seems some rust -> python bridging issue), so I reverted them